### PR TITLE
Correct abs links

### PR DESCRIPTION
**Pull request by mnazzaro**

### CODE REVIEW

File: browse/templates/abs/extra_services.html [hunk 1]
```diff
@@ -185,14 +181,14 @@ <h2>Access Paper:</h2>
       <div class="abs-license">
         {%- if withdrawn -%}
         <div hidden>No license for this version due to withdrawn</div>
+        {%- elif abs_meta.license.icon_uri_path -%}
+        <a href="{{abs_meta.license.effective_uri}}" title="Rights to this article">
+          <img src="https://arxiv.org{{ abs_meta.license.icon_uri_path }}"/>
```
Comment (SBBCornell) [inline browse/templates/abs/extra_services.html:186]:
```suggestion
          <img alt="license icon" role="presentation" src="https://arxiv.org{{ abs_meta.license.icon_uri_path }}"/>
```

File: browse/templates/abs/extra_services.html [hunk 2]
```diff
@@ -56,14 +56,10 @@
       {%- endif -%}
       {%- if format == 'src' and current-%}
   <li><a href="{{url_for('src.src', arxiv_id=requested_id)}}" class="abs-button download-eprint">TeX Source</a></li>
-      {%- elif format.startswith('ps') -%}
```
Comment (bdc34) [inline browse/templates/abs/extra_services.html:59]: As far as I can tell from ARXIVCE-1159 this PR isn't suppose to remove the 'ps' format from the list.

File: browse/templates/abs/extra_services.html [hunk 3]
```diff
@@ -56,14 +56,10 @@
...
-  <li><a href="{{url_for('dissemination.ps', arxiv_id=requested_id)}}" class="abs-button download-ps">PostScript{% if format.endswith('(400)') %} (400 dpi){% elif format.endswith('(600)') %} (600 dpi){% elif format.endswith('(cm)') %} (Type I cm){% elif format.endswith('(CM)') %} (Type I CM){% endif %}</a></li>
-      {%- elif format == 'dvi' -%}
-  <li><a href="{{url_for('dissemination.dvi', arxiv_id=requested_id)}}" class="abs-button download-dvi">{{ format.upper() }}</a></li>
       {%- elif format == 'html' -%}
   <li><a href="{{url_for('dissemination.html', arxiv_id=requested_id)}}" accesskey="f" class="abs-button download-html">{{ format.upper() }}</a></li>
-      {%- elif format == 'nops' -%}
```
Comment (bdc34) [inline browse/templates/abs/extra_services.html:65]: As far as I can tell from ARXIVCE-1159 this PR isn't suppose to remove the 'nops' format from the list.

File: browse/templates/abs/extra_services.html [hunk 4]
```diff
@@ -185,14 +181,14 @@ <h2>Access Paper:</h2>
       <div class="abs-license">
         {%- if withdrawn -%}
         <div hidden>No license for this version due to withdrawn</div>
+        {%- elif abs_meta.license.icon_uri_path -%}
+        <a href="{{abs_meta.license.effective_uri}}" title="Rights to this article">
+          <img src="https://arxiv.org{{ abs_meta.license.icon_uri_path }}"/>
+          <span>view license</span>
+        </a>
         {%- else -%}
-          {%- if 'other' in formats -%}
```
Comment (bdc34) [inline browse/templates/abs/extra_services.html:189]: :+1:

**Review by bdc34 — CHANGES_REQUESTED** [review]
I'm not sure if this is intended to alter 'ps' and 'nops' display in the download area.

File: browse/templates/abs/extra_services.html (hunk 2 above)
Comment (bdc34) [reply browse/templates/abs/extra_services.html:59]: Mark tells me this PR is intended to remove the PS download.

File: browse/templates/abs/extra_services.html (hunk 3 above)
Comment (bdc34) [reply browse/templates/abs/extra_services.html:65]: Mark tells me this PR is intended to remove the 'nops' download link.

File: browse/templates/abs/extra_services.html (hunk 2 above)
Comment (SBBCornell) [reply browse/templates/abs/extra_services.html:59]: We just had a quick chat (me and Mark) and were in agreement that PS should be found on the /formats page, but not on this main abs list.  Same with DVI.

**Review by bdc34 — APPROVED** [review]

File: browse/templates/abs/extra_services.html (hunk 2 above)
Comment (bdc34) [reply browse/templates/abs/extra_services.html:59]: @jimentwood The PS download links on the abs page have been changed.

**Pull request merged**